### DISCUSSION TITLE
Failed to invite user: failed to make invite request: duplicate key value violates unique constraint "profiles_pkey"

### DISCUSSION THREAD

**Discussion by champec** [supabase/supabase#12704] [Questions] [answered]
I am trying to invite a new user from the supabse website - for testig purposes and i am getting the following error.

Failed to invite user: failed to make invite request: duplicate key value violates unique constraint "profiles_pkey"

1. What does it mean
2.  How can i get more information about the cause of this error, is there an error log somehwere ?

I only have one table a profiles table and you the fault sql function to create a profile when the user signs up

**Comment by GaryAustin1** [comment, marked as answer]
Seems to be saying you are trying to insert to the profiles table twice.  My guess is you have two insert trigger functions on auth.users and so each runs with the same UUID and tries to insert into the profiles table.

You can run this query and see if there are two triggers on users table...

```
SELECT  event_object_table AS table_name ,trigger_name         
FROM information_schema.triggers  
GROUP BY table_name , trigger_name 
ORDER BY table_name ,trigger_name 
```

**Reply by champec** [reply]
I forgot to comment I basically worked it out and it was exactly as you had put it. I ran a some sql functions to remove the constraints only to find that it created 2 rows of users. Silly mistake but I learnt a lot about trigger functions and the sql box in the supabase dashboard
